src/esw/brushless.py

```diff
@@ -212,13 +218,108 @@ async def update(self) -> None:
             await self._connect()
 
 
-class DriveApp:
+class MotorsManager:
     BASESTATION_TO_ROVER_NODE_WATCHDOG_TIMEOUT_S = 1
-    DRIVE_NAMES = ["FrontLeft", "FrontRight", "MiddleLeft", "MiddleRight", "BackLeft", "BackRight"]
 
+    def __init__(self, motor_controller_info_by_name, transport, publish_topic: str):
```

Comment:
can we add the type declarations to motor_controller_info_by_name and transport?

Reply:
so the issue is that I'm not actually sure what the type declarations of the two are because it is part of the library, i could maybe try to do some research tho